ppx/test/keys.t/run.t

```diff
@@ -110,7 +110,7 @@ __^
 
 <img src={author.Author.imageUrl} />
 ______^
-
+TODO: This is a regression, the type is not correct should be a string
```

Comment:
I also caught this in one of my PRs. do you know if this is this related to the ppxlib upgrade? or have you not identified a cause yet?

Reply:
Didn't get deep into it, but want to revisit the locations of props/elements

Reply:
My plan now is to merge this, the React 19 and another PR investigating the locations